### DISCUSSION TITLE
[Problem] Can´t donwload the App from the PlayStore

### DISCUSSION THREAD

**Discussion by Lunasciel** [KRTirtho/spotube#1586] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I can´t donwload the App from the PlayStore

### Expected Behavior

Donwload the App

### Steps to reproduce

1. Open PlayStore
2. Search for Spotube
3. Click on install
4. Installation Fails

### Operating System

Andriod

### Installation source

Play Store (Android)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Which version of Android are you using? Do you have enough storage?

**Reply by Lunasciel** [reply]
I am using Andriod 14 and yes enough storage. ^^

**Comment by Lunasciel** [comment]
It still don't work.
